orchagent/portsorch.cpp

```diff
@@ -735,60 +731,65 @@ bool PortsOrch::bindAclTable(sai_object_id_t id, sai_object_id_t table_oid, sai_
             port.m_egress_acl_table_group_id = groupOid;
         }
 
+        setPort(port.m_alias, port);
```

Comment:
Should we save the information after binding successfully?

Reply:
ok,  we can save the information here.  But we may need some way to indicate whether the binding was successful or not.  Currently, in many places we ignored the bind() function results.  We need fix those places.  We should address it if not in this PR.